src/Files.App/Data/Contexts/ContentPage/ContentPageContext.cs

```diff
@@ -43,7 +43,7 @@ internal sealed class ContentPageContext : ObservableObject, IContentPageContext
 
 		public bool CanCreateItem => GetCanCreateItem();
 
-		public bool IsMultiPaneEnabled => ShellPage is not null && ShellPage.PaneHolder is not null && ShellPage.PaneHolder.IsMultiPaneEnabled;
+		public bool IsMultiPaneEnabled => ShellPage is not null && ShellPage.PaneHolder is not null && ShellPage.PaneHolder.CanBeDualPane;
```

Comment:
```suggestion
		public bool IsMultiPaneAvailable => ShellPage is not null && ShellPage.PaneHolder is not null && ShellPage.PaneHolder.CanBeDualPane;
```

Reply:
I used to want it to be CanBeDualPane but this makes more sense.
Renamed.